src/components/BlockingViews/ForceFullScreenView/index.native.tsx

```diff
@@ -0,0 +1,9 @@
+import type ForceFullScreenViewProps from './types';
+
+function ForceFullScreenView({children}: ForceFullScreenViewProps) {
+    return children;
```

Comment:
why do we need to use a component to apply full screen style depending on the platform? why can't we just use different style?